docs/project.md

```diff
@@ -3,3 +3,12 @@ sidebar_position: 3
 ---
 
 # Project
+
```

Comment:
```suggestion
The purpose of the project is to build a hardware device that runs software written in Rust.

## Deliverables
- documentation - we will provide a template `README.md` file in the template repository
  - requirements (hardware and software)
  - hardware design - use fritzing, tinkercad (if the board exists) or an EDA to design it
  - software design
    - detail design
    - functional diagram
- software

## Hardware

... hardware rules

## Software

It has to run software written in Rust. Students can you:
- [embassy-rs](...)
- [RTIC]()
- [bare metal](..) - the cortex-r-rt crate
- [Tock](https://www.tockos.org)
- any other software framework that is written in Rust

```

## Example Projects

## F.A.Q
*Q:* Can I use ...?
*A:*: ...

*Q:* Can I use ...?
*A:*: ...

*Q:* Can I use ...?
*A:*: ...

...

This is a suggestion on how to structure the document